config/crd/kustomization.yaml

```diff
@@ -1,8 +1,17 @@
-apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
 - bases/postgres-operator.crunchydata.com_crunchybridgeclusters.yaml
 - bases/postgres-operator.crunchydata.com_postgresclusters.yaml
 - bases/postgres-operator.crunchydata.com_pgupgrades.yaml
 - bases/postgres-operator.crunchydata.com_pgadmins.yaml
+
+patches:
+- target:
+    kind: CustomResourceDefinition
+  patch: |-
+    - op: add
+      path: /metadata/labels
+      value:
+        app.kubernetes.io/name: pgo
```

Comment:
:question: I like moving these labels to one location, but I am curious if this changes "what" the CRD is.

To hopefully explain that better, it seems the CRD was a single file (`config/crd/bases/postgres-operator.crunchydata.com_postgresclusters.yaml`) and now it's the base file + this patch. Does that sound right?

Reply:
Yes, it does a bit. These files in `config/crd/bases` differ, but this `config/crd` [kustomization](https://kubectl.docs.kubernetes.io/references/kustomize/glossary/#kustomization) produces the same things as before. The remaining Make targets and scripts use the kustomization.